apps/admin/pages/news/index.tsx

```diff
@@ -126,6 +135,18 @@ const NewsPage = () => {
         ),
       }}
     >
+      <ButtonGroup overflowX="auto">
```

Comment:
We can show small buttons maybe?

```suggestion
      <ButtonGroup size={'sm'}>
```